### PR TITLE
Correct style using rubocop (part 3)

### PR DESCRIPTION
**Pull request by astrozzc**
Cops used:
Naming/PredicateName

Style/BlockDelimiters
Style/BracesAroundHashParameters
Style/CommentedKeyword
Style/ConditionalAssignment
Style/ExpandPathArgument
Style/IdenticalConditionalBranches
Style/IfInsideElse
Style/InverseMethods
Style/MethodCallWithArgsParentheses
Style/ParenthesesAroundCondition
Style/RedundantParentheses
Style/SpecialGlobalVars
Style/TrailingCommaInArguments
Style/UnlessElse
Style/RedundantReturn

@miq-bot add_label ivanchuk/no, refactoring

### CODE REVIEW

**Comment by astrozzc** [comment]
@miq-bot add_label ivanchuk/no, refactoring
@miq-bot assign @tinaafitz 
@miq-bot add_reviewer @gmcculloug

File: app/models/miq_ae_datastore.rb [hunk 1]
```diff
@@ -271,4 +271,5 @@ def self.path_includes_domain?(path, options = {})
     nsd, = ::MiqAeEngine::MiqAePath.split(path, options)
     MiqAeNamespace.find_by_fqname(nsd, false) != nil
   end
-end # module MiqAeDatastore
+end
+# module MiqAeDatastore
```
Comment (gmcculloug) [inline app/models/miq_ae_datastore.rb]: This comment is not needed, we can just delete it.

File: app/models/miq_ae_yaml_import.rb [hunk 2]
```diff
@@ -321,4 +321,5 @@ def update_attributes(domain_obj)
   def domain_locked?(domain_name)
     MiqAeDomain.find_by(:name => domain_name)&.contents_locked? ? true : false
   end
-end # class
+end
+# class
```
Comment (gmcculloug) [inline app/models/miq_ae_yaml_import.rb]: Same with this comment.

File: lib/miq_automation_engine/engine/miq_ae_engine/miq_ae_object.rb [hunk 3]
```diff
@@ -192,11 +192,11 @@ def state_machine?
...
-      when 'MiqAePassword'            then xml.Password OPAQUE_PASSWORD
-      when 'String'                   then xml.String   value
-      when 'Fixnum'                   then xml.Fixnum   value
-      when 'Symbol'                   then xml.Symbol   value.to_s
-      when 'TrueClass', 'FalseClass'  then xml.Boolean  value.to_s
+      when 'MiqAePassword'            then xml.Password(OPAQUE_PASSWORD)
+      when 'String'                   then xml.String(  value)
+      when 'Fixnum'                   then xml.Fixnum(  value)
+      when 'Symbol'                   then xml.Symbol(  value.to_s)
+      when 'TrueClass', 'FalseClass'  then xml.Boolean( value.to_s)
```
Comment (gmcculloug) [inline lib/miq_automation_engine/engine/miq_ae_engine/miq_ae_object.rb]: @astrozzc Please cleanup the leading whitespace.

Comment (astrozzc) [reply lib/miq_automation_engine/engine/miq_ae_engine/miq_ae_object.rb]: The YAML.safe_load changes are removed from this PR. And the leading whitespaces are fixed.

File: manageiq-automation_engine.gemspec [hunk 4]
```diff
@@ -15,8 +15,8 @@ Gem::Specification.new do |s|
 
   s.files = Dir["{app,lib}/**/*", "LICENSE.txt", "Rakefile", "README.md"]
 
-  s.add_dependency "rubyzip", "~>1.2.2"
+  s.add_dependency("rubyzip", "~>1.2.2")
 
-  s.add_development_dependency "codeclimate-test-reporter", "~> 1.0.0"
-  s.add_development_dependency "simplecov"
+  s.add_development_dependency("codeclimate-test-reporter", "~> 1.0.0")
+  s.add_development_dependency("simplecov")
```
Comment (gmcculloug) [inline manageiq-automation_engine.gemspec]: @astrozzc This is usually treated as a DSL so we can leave it unchanged.

Would be good to look at adding `add_dependency`, `add_development_dependency` and even `add_runtime_dependency` to the robocop `IgnoredMethods` list here: https://github.com/ManageIQ/guides/blob/master/.rubocop_base.yml#L94

**Pull request merged**